### PR TITLE
New Jewelry (and two new spiritual items) for Black History Month

### PR DESCRIPTION
**Pull request by 7erracotta**
#### Summary
Content "Adds new jewelry for black history month, such as cowrie necklaces and various types of hair beads."

#### Purpose of change
I wanted to increase the representation of African fashion and culture within C: DDA.

#### Describe the solution
There's now a chance the player can find hair beads as well as cowrie bracelets and necklaces, which the player can meditate with.

#### Describe alternatives you've considered
N/A

#### Testing
Spawned in, opened the debug menu, and tested to see if the items would spawn in the item groups I assigned them, which they did. Spawned in as a character with the spiritual trait, spawned a cowrie necklace then meditated with it, resulting in a small mood buff.

#### Additional context
"N/A"

### CODE REVIEW

File: data/json/items/armor/jewelry.json [hunk 1]
```diff
@@ -94,6 +94,34 @@
     "color": "brown",
     "flags": [ "NO_WEAR_EFFECT" ]
   },
+  {
+    "id": "cowrie_bracelet",
+    "copy-from": "bead_bracelet",
+    "type": "ARMOR",
+    "name": { "str": "cowrie bracelet" },
+    "description": "a bracelet made of cowrie shells, valued in many african cultures such as the Yoruba. If you want, you could meditate with this.",
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:102]:
```suggestion
    "description": "A bracelet made of cowrie shells, valued in many African cultures such as the Yoruba.  If you want, you could meditate with this.",
```

File: data/json/items/armor/jewelry.json [hunk 2]
```diff
@@ -94,6 +94,34 @@
...
+    "material": [ "bone" ],
+    "color": "white"
+  },
+  {
+    "id": "cowrie_hair_beads",
+    "copy-from": "wooden_hair_beads",
+    "type": "ARMOR",
+    "name": { "str": "cowrie hair beads" },
+    "description": "an assortment of cowrie shells used to decorate locs or braids.",
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:111]:
```suggestion
    "description": "An assortment of cowrie shells used to decorate locs or braids.",
```

File: data/json/items/armor/jewelry.json [hunk 3]
```diff
@@ -94,6 +94,34 @@
...
+    "material": [ "bone" ],
+    "color": "white",
+    "flags": [ "NO_WEAR_EFFECT", "FANCY" ]
+  },
+  {
+    "id": "cowrie_necklace",
+    "copy-from": "bead_necklace",
+    "type": "ARMOR",
+    "name": { "str": "cowrie necklace" },
+    "description": "a necklace made of cowrie shells, valued in many african cultures such as the Yoruba. If you want, you could meditate with this.",
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:121]:
```suggestion
    "description": "A necklace made of cowrie shells, valued in many African cultures such as the Yoruba.  If you want, you could meditate with this.",
```

File: data/json/items/armor/jewelry.json [hunk 4]
```diff
@@ -94,6 +94,34 @@
...
+    "name": { "str": "cowrie bracelet" },
+    "description": "a bracelet made of cowrie shells, valued in many african cultures such as the Yoruba. If you want, you could meditate with this.",
+    "material": [ "bone" ],
+    "color": "white"
+  },
+  {
+    "id": "cowrie_hair_beads",
+    "copy-from": "wooden_hair_beads",
+    "type": "ARMOR",
+    "name": { "str": "cowrie hair beads" },
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:110]:
```suggestion
    "name": { "str": "cowrie hair beads", "str_pl": "sets of cowrie hair beads" },
```

File: data/json/items/armor/jewelry.json [hunk 5]
```diff
@@ -1123,6 +1151,17 @@
     "use_action": [ "MEDITATE" ],
     "flags": [ "NO_WEAR_EFFECT" ]
   },
+  {
+    "id": "wooden_hair_beads",
+    "copy-from": "garnet_silver_tiara",
+    "type": "ARMOR",
+    "name": { "str": "wooden hair beads" },
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:1158]:
```suggestion
    "name": { "str": "wooden hair beads", "str_pl": "sets of wooden hair beads" },
```
If you don't like "sets of" and can't think of anything else you can make it `"name": { "str_sp": "wooden hair beads" }` instead

File: data/json/items/armor/jewelry.json [hunk 6]
```diff
@@ -4025,6 +4064,16 @@
     "description": "A shiny, gold tiara adorned with onyx.",
     "color": "dark_gray"
   },
+  {
+    "id": "gold_hair_beads",
+    "copy-from": "garnet_gold_tiara",
+    "type": "ARMOR",
+    "name": { "str": "gold hair beads" },
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:4071]:
```suggestion
    "name": { "str": "gold hair beads", "str_pl": "sets of gold hair beads" },
```

File: data/json/items/armor/jewelry.json [hunk 7]
```diff
@@ -4177,5 +4226,16 @@
       { "id": "dark_tiara", "name": { "str": "dark tiara" }, "description": "A dark silver tiara adorned with onyx." }
     ],
     "color": "dark_gray"
+  },
+  {
+    "id": "silver_hair_beads",
+    "copy-from": "garnet_silver_tiara",
+    "type": "ARMOR",
+    "name": { "str": "silver hair beads" },
```
Comment (Procyonae) [inline data/json/items/armor/jewelry.json:4234]:
```suggestion
    "name": { "str": "silver hair beads", "str_pl": "sets of silver hair beads" },
```

**Review by Procyonae — CHANGES_REQUESTED** [review]
The translation software needs 2 spaces after sentences, descriptions typically start with capitals and if a name can't be pluralised by putting an s on the end it needs manually specifying.

File: data/json/items/armor/jewelry.json [hunk 8]
```diff
@@ -1123,6 +1151,17 @@
     "use_action": [ "MEDITATE" ],
     "flags": [ "NO_WEAR_EFFECT" ]
   },
+  {
+    "id": "wooden_hair_beads",
+    "copy-from": "garnet_silver_tiara",
+    "type": "ARMOR",
+    "name": { "str": "wooden hair beads", "str_pl": "sets of wooden hair beads" },
+    "description": "An assortment of wooden beads used to decorate locs or braids.",
+    "material": [ "wood" ],
```
Comment (Maleclypse) [inline data/json/items/armor/jewelry.json:1160]: Do you need to adjust weight and volume? I imagine a metal tiara weighs and takes more volume than hair beads.

File: data/json/items/armor/jewelry.json (hunk 6 above)
Comment (Maleclypse) [inline data/json/items/armor/jewelry.json:4071]:
```suggestion
    "name": { "str_sp": "gold hair beads" },
```

File: data/json/items/armor/jewelry.json (hunk 8 above)
Comment (7erracotta) [reply data/json/items/armor/jewelry.json:1160]: Might need to, That was an oversight on my part, apologies!

**Pull request merged**